### PR TITLE
Issues resolved 

### LINKED ISSUE #1: Changes needed for final deployment
### In order to deploy the game on the final website, the following changes are needed.
Make the changes in a new draft PR titled `Final deployment prep` to this repo and add `Fixes #1` as its description.

## Changes requested
- [x] Use `(await import(url)).default` as the image source instead fetching the images or using image urls
- [x] Remove unnecessary components like https://github.com/TechForAutismDyslexia/Sentence-Verification-Global/blob/6a180e000e8d7ac13aa39811967be940e885a8dd/src/components/ConfettiComponent.js#L4-L6
- [x] Use your game icon as favicon
- [x] Rename all react components or files to have `.jsx` extension
- [x] Move all jsons to `/src/assets`
- [x] Remove all commented code in files
- [x] Make the root path of the game as `/games/sentenceverification1`
- [x] Do not use `require()` in your code

### PR DESCRIPTION
**Pull request by sumithkumar123**
Fixed #1

### CODE REVIEW

File: src/setupTests.jsx
Comment (Jai-JAP) [inline src/setupTests.jsx:1]: revert this file name as this doesn't return any react components.

File: public/index.html
Comment (Jai-JAP) [inline public/index.html:1]: Remove the extra <!DOCTYPE html> and fix this file correctly

**Review by Jai-JAP — CHANGES_REQUESTED** [review]

File: public/index.html
Comment (Jai-JAP) [inline public/index.html:1]: Remove the extra <!DOCTYPE html> in this file and verify all your fixed properly

**Review by Jai-JAP — CHANGES_REQUESTED** [review]

File: public/index.html [hunk 1]
```diff
@@ -1,63 +1,25 @@
...
+  
+    <title>Sentence Verification Global</title>
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="../src/components/styles.css">
 
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (Jai-JAP) [inline public/index.html:32]: You need this for all react apps.

**Review by Jai-JAP — APPROVED** [review]

**Review by brohithkr — APPROVED** [review]

**Pull request merged**